MoviesExample/core/src/main/java/com/novoda/demo/movies/api/MoviesApi.java

```diff
@@ -14,5 +13,5 @@
     Call<MoviesResponse> topRated(@Query("page") int page);
 
     @GET("movie/{id}/videos?api_key="+API_KEY)
-    Single<VideosResponse> videos(@Path("id") String movieId);
+    Call<VideosResponse> videos(@Path("id") String movieId);
```

Comment:
Call handles moving stuff to a bg thread?

Reply:
Yes, `Call` has two methods to run the query: `execute` (sync) and `enqueue` (async)

Reply:
noice